src/ios/SimpleCameraPreview.m

```diff
@@ -57,19 +57,20 @@ - (void) disable:(CDVInvokedUrlCommand*)command {
             for(AVCaptureOutput *output in self.sessionManager.session.outputs) {
                 [self.sessionManager.session removeOutput:output];
             }
-            [self.sessionManager.session stopRunning];
-            self.sessionManager = nil;
             dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
@zafirskthelifehacker est-ce que tu peux faire des tests sans le dispatch_async ? Et voir le résultat sur la performance de l'app ? Ça peut être dans une 2ème PR qui part de master et on laisse l'ordre comme c'est mais on enlève juste le dispatch_async.

Reply:
Cannot remove `dispatch_async` as removeFromSuperview requires to run asynchronous. Added Callback in `dispatch_async`.